package.json

```diff
@@ -46,7 +46,7 @@
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
     "jest-junit": "^16.0.0",
-    "lodash": "^4.17.21",
+    "lodash-es": "^4.17.21",
```

Comment:
we only use this in tests and this one works in naked ESM fine without more build plugins